packages/react/src/ReactElementValidator.js

```diff
@@ -214,14 +214,24 @@ function validateChildKeys(node, parentType) {
  *
  * @param {ReactElement} element
  */
+var propTypesMisspellWarningShown;
 function validatePropTypes(element) {
   var componentClass = element.type;
   if (typeof componentClass !== 'function') {
     return;
   }
   var name = componentClass.displayName || componentClass.name;
   var propTypes = componentClass.propTypes;
-
+  if (
```

Comment:
Maybe let's put it into an `else` clause to the next `if`?

```js
if (propTypes) {
  // ...
} else if (componentClass.PropTypes && !propTypesMisspellWarningShown) {
  // ...
}
```

Reply:
I have also tested this `fixtures/dom` by introducing some props and misspelling `propTypes`